Moderation-Policy.md

```diff
@@ -1,154 +1,306 @@
-* Only Collaborators with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
-* The TSC serves as the final arbiter for all Moderation issues (see: [Escalation to the TSC](#escalation-to-the-tsc)).
-* Only a TSC member may Remove or Ban an individual from the Node.js GitHub Organization.
-* For any Removal or Banning action, an issue describing the reasons for the action, and identifying the Github account being acted upon, must be posted to the Moderation Repository with an explanation provided by the TSC member performing the action.
-* Any individual Banned from the Node.js GitHub Organization will be recommended for exclusion from any Node.js Foundation sponsored event or activity.
-* Minor edits to the formatting of a Post or to correct typographical errors are not considered to be "Moderation". Such edits and their intent should still be documented with a short note indicating who made the edit and why.
-
-#### Collaborator Posts
-
-* Collaborators must not Moderate any Post authored by another Collaborator without first giving the author at least 24 hours (from the time of the initial request) to modify or remove the Post on their own.
-* If the author of the Post disagrees that Moderation is required, the matter can be [escalated to the TSC](#escalation-to-the-tsc) for resolution. In such cases, no Moderation action should be taken until a decision by the TSC is made.
-* In extreme circumstances involving either obvious gross violations of the Node.js [Code of Conduct][] or possible compromise of a Collaborator's GitHub account, the TSC can be consulted to waive the 24 hour grace period and dispute process.
-* When Moderating any Post authored by another Collaborator, the moderating Collaborator must:
+* Any Collaborator with commit rights to a given repository may Moderate Posts
+  within that repository's issue tracker.
+* The Moderation Team serves as the final arbiter for all Moderation issues.
+* Moderation Team members may Remove or Ban an individual from the Node.js
+  GitHub Organization.
+* For any Removal or Banning action, an issue describing the reasons for the
+  action, and identifying the Github account being acted upon, must be posted
+  to the Moderation Repository with an explanation provided by the Moderation
+  Team member performing the action.
+* Any individual Banned from the Node.js GitHub Organization will be recommended
+  for exclusion from any Node.js Foundation sponsored event or activity.
+* Minor edits to the formatting of a Post or to correct typographical errors
+  are not considered to be "Moderation". Such edits and their intent should
+  still be documented with a short note indicating who made the edit and why.
+
+### Collaborator Posts
+
+* Prior to Moderating any Post authored by a Collaborator, the author must be
+  given a reasonable opportunity to modify or remove the Post on their own.
```

Comment:
The original text gave 24 hours. The feedback that led to this change suggested that in some cases, 24 hours was too much time. 48 would definitely be too much time.